smurfs/src/components/Smurfs.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { fetchSmurfs } from '../actions'
+
+class Smurfs extends React.Component {
+    constructor() {
+        super();
+    }
+    componentDidMount() {
+        this.props.fetchSmurfs();
+    }
+
+    render() {
+        return (
+            <div>
+                {this.props.smurfs.map(smurf =>
+                    <div>
+                        <h1>{smurf.name}</h1>
+                        <h2>{smurf.age}</h2>
+                        <h2>{smurf.height}</h2>
+                    </div>
+                )}
+            </div>
+        )
+    }
+}
+
+const mapStateToProps = (state) => {
+    return {
+        smurfs: state.smurfs,
+        fetchingSmurfs: state.fetchingSmurfs
```

Comment:
you are getting fetchingSmurfs from the store but you are not using it....something you forgot to implement? Maybe a spinner?